Makefile

```diff
@@ -116,18 +116,15 @@ generate-keys: $(BUILD_FOLDER)
 .PHONY: build docs
 build: lint unit-test build-minimal build-iso-generator
 
-build-all: build-image build-minimal-assisted-iso-generator-image
+build-all: build-image build-minimal-assisted-iso-generator-image build-onprem
 
 build-minimal: $(BUILD_FOLDER)
 	CGO_ENABLED=0 go build -o $(BUILD_FOLDER)/assisted-service cmd/main.go
 
 build-iso-generator: $(BUILD_FOLDER)
 	CGO_ENABLED=0 go build -o $(BUILD_FOLDER)/assisted-iso-create assisted-iso-create/main.go
 
-build-onprem-dependencies: 
-	skipper make build-all
-
-build-onprem: build-onprem-dependencies
+build-onprem:
```

Comment:
build-onprem is meant to run outside of skipper and thus outside of the docker container. The next couple of lines, where it does "podman pull", pulls the assisted-service and assisted-iso-create container images from docker and places them in the podman data store. They are then used as sources for the assisted-service-onprem image build.

Reply:
If podman is available on the Jenkins workers, then may I suggest doing it this way: https://github.com/rwsu/assisted-service/commit/6a4afe6613618545b4b244f02e291e0d09b65baa